src/ButtonGroup.js

```diff
@@ -9,35 +9,53 @@ var ButtonGroup = function(ctx, x, y, buttons) {
 
 ButtonGroup.prototype = Object.create(Phaser.Group.prototype);
 ButtonGroup.prototype.constructor = ButtonGroup;
+
 ButtonGroup.prototype.makeButton = function(ctx, btn) {
+    var name = btn.name || '';
     var x = btn.x || 0;
     var y = btn.y || 0;
     var size = btn.size || 25;
     var anchor = btn.anchor || [0.5, 0.5];
```

Comment:
I feel like these `||` commands are dangerous, if the value is nothing there someone probably made a mistake. It maybe hard to track the bug if we set it otherwise here.